code/__DEFINES/tools.dm

```diff
@@ -8,6 +8,7 @@
 #define TOOL_ANALYZER "analyzer"
 #define TOOL_MINING "mining"
 #define TOOL_SHOVEL "shovel"
+#define TOOL_DRAPES "surgicaldrapes"
```

Comment:
Wait why does this tool behavior even exist, drape behavior is entirely handled by a component

Reply:
So it's mostly for the radial menu for the omnitool. It needs to display the tool name(which is the tool behavior) along with its icon.

It is redundant with the component but it does not conflict with its functionality so it won't cause any issues I promise